build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
     dependencies {
         // If you ever see strange Gradle behavior that looks related to the version,
         // double check that this matches build.gradle.kts
-        classpath 'com.android.tools.build:gradle:3.3.1'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```

Comment:
This version should be the same as the one used in `buildSrc/build.gradle.kts`.  Can you bump that one too?